app/queries/schools_query.rb

```diff
@@ -35,6 +37,33 @@ def organisation_ids_near_location(location_coordinates)
     ).map(&:id)
   end
 
+  def schools_to_show_condition(scope)
+    return scope if filter_params[:schools_to_show] == "all"
+
+    scope.where.associated(:placement_preferences)
+  end
+
+  def itt_statuses_condition(scope)
+    hosting_interests = filter_params[:itt_statuses]
+    return scope if hosting_interests.blank?
+
+    conditions = []
+
+    if hosting_interests.include?("open")
+      conditions << scope.where(placement_preferences: { appetite: "interested" })
+    end
+
+    if hosting_interests.include?("not_open")
+      conditions << scope.where(placement_preferences: { appetite: "not_open" })
+    end
+
+    if hosting_interests.include?("actively_looking")
+      conditions << scope.where(placement_preferences: { appetite: "actively_looking" })
+    end
+
+    conditions.reduce(scope.none) { |combined_scope, condition| combined_scope.or(condition) }
+  end
```

Comment:
This doesn't seem like the simplest solution.
I would consider changing the value of the options passed into the filter, making them the expected enum values.
```suggestion
  def itt_statuses_condition(scope)
    return scope if filter_params[:itt_statuses].blank?
    
    scope.where(placement_preferences: { appetite: filter_params[:itt_statuses] })
    end
```